packages/x-data-grid/src/components/GridRow.tsx

```diff
@@ -56,7 +56,6 @@ export interface GridRowProps extends React.HTMLAttributes<HTMLDivElement> {
    */
   index: number;
   rowHeight: number | 'auto';
-  offsetTop: number | undefined;
```

Comment:
I think I left this prop there to be able to play with the CSS layout method (e.g. switch to `transform: translate()` on each row) without introducing a breaking change (this slot is exposed publicly), but I guess it's fine to remove it as we don't plan to do any change.